include/revng/ABI/ModelAlignment.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/ADT/RecursiveCoroutine.h"
+#include "revng/Model/Type.h"
+#include "revng/Model/VerifyHelper.h"
+
+namespace abi {
+
+RecursiveCoroutine<std::optional<uint64_t>>
+naturalAlignment(model::VerifyHelper &VH,
+                 const model::QualifiedType &Type,
+                 model::ABI::Values ABI);
+
+/// \brief Compute the natural alignment of the type as expected by the selected
+///        ABI when the type is passed as a parameter.
```

Comment:
`\brief` is deprecated.

Reply:
vscode starting suggesting those, so I just went with the flow. But sure, I'll drop them.

Reply:
Dropped those.

Reply:
Whoops, it seems like I missed one, thanks for catching it.

Reply:
Now it's gone for sure.